lms/djangoapps/courseware/testutils.py

```diff
@@ -90,27 +111,54 @@ def setup_user(self, admin=False, enroll=False, login=False):
         if login:
             self.login()
 
-    def verify_response(self, expected_response_code=200, url_params=None):
+    def set_block_to_be_requested(self, block_name):
```

Comment:
@nasthagiri: I had this as `block_to_be_tested`, but Python was treating this as a test and giving an error because of the parameter.

Would you mind reviewing this test code when you get a chance?